.ocularrc.js

```diff
@@ -9,17 +9,12 @@ const testDir = join(packageRoot, 'test');
 
 /** @type {OcularConfig} */
 const config = {
-  babel: false,
 
   lint: {
-    paths: ['modules', 'docs', 'test'], // 'examples'], module resolution errors
+    paths: ['modules', 'test'], // 'docs', 'examples'], module resolution errors
```

Comment:
I thought we had markdown linter?

Reply:
We do, but it was not used as `.md` was not listed in `extensions`. We can enable it in another PR it will likely result in a bunch of prettier changes.

Reply:
Sounds good. We probably need to add `mdx` as well